diff -uNdr lorax-0.3.2.orig/src/pylorax/installtree.py lorax-0.3.2/src/pylorax/installtree.py
--- lorax-0.3.2.orig/src/pylorax/installtree.py	2011-03-21 16:50:33.000000000 +0300
+++ lorax-0.3.2/src/pylorax/installtree.py	2011-03-23 00:42:34.724960775 +0300
@@ -36,6 +36,7 @@
 import constants
 from sysutils import *
 
+from rfrinstalltree import RFRInstallTree
 
 class LoraxInstallTree(BaseLoraxClass):
 
@@ -373,7 +374,24 @@
     def move_repos(self):
         src = joinpaths(self.root, "etc/yum.repos.d")
         dst = joinpaths(self.root, "etc/anaconda.repos.d")
-        shutil.move(src, dst)
+        # hacked russianfedora ;)
+        if os.path.exists(src):
+            shutil.rmtree(src, True)
+        
+        rfr_dst = joinpaths(dst, 'russianfedoraremix-release-anaconda.repo')
+        gnome_cfg_dst = joinpaths(self.root, "etc/gnome.cfg")
+        kde_cfg_dst = joinpaths(self.root, "etc/kde.cfg")
+        xfce_cfg_dst = joinpaths(self.root, "etc/xfce.cfg")
+        lxde_cfg_dst = joinpaths(self.root, "etc/lxde.cfg")
+        tigro_cfg_dst = joinpaths(self.root, "etc/tigro.cfg")
+
+        rfr = RFRInstallTree()
+        rfr.crete_rfr_repofile(rfr_dst)
+        rfr.create_gnome_cfg(gnome_cfg_dst)
+        rfr.create_kde_cfg(kde_cfg_dst)
+        rfr.create_xfce_cfg(xfce_cfg_dst)
+        rfr.create_lxde_cfg(lxde_cfg_dst)
+        rfr.create_tigro_cfg(tigro_cfg_dst)
 
     def create_depmod_conf(self):
         text = "search updates built-in\n"
diff -uNdr lorax-0.3.2.orig/src/pylorax/rfrinstalltree.py lorax-0.3.2/src/pylorax/rfrinstalltree.py
--- lorax-0.3.2.orig/src/pylorax/rfrinstalltree.py	1970-01-01 03:00:00.000000000 +0300
+++ lorax-0.3.2/src/pylorax/rfrinstalltree.py	2011-03-23 00:38:11.228693678 +0300
@@ -0,0 +1,258 @@
+from base import BaseLoraxClass
+
+class RFRInstallTree(BaseLoraxClass):
+    def __init__(self):
+        BaseLoraxClass.__init__(self)
+
+    def _write_content_to_file(self, filename, content):
+        try:
+            somefile = open(filename, 'w')
+        except:
+            self.perror("Error, file %s not open!" % filename)
+            return False
+        
+        somefile.write(content)
+        somefile.close()
+
+        return True
+
+    def create_rfr_repofile(self, filename):
+        repofile_content = """
+[russianfedora]
+name=RFRemix \$releasever - \$basearch
+failovermethod=priority
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=build-russianfedora-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=0
+
+[russianfedora-updates]
+name=RFRemix \$releasever - \$basearch - Updates
+failovermethod=priority
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=build-russianfedora-updates-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=0
+
+[rpmfusion-free-russianfedora]
+name=RPM Fusion for Fedora \$releasever - Free
+failovermethod=priority
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=build-rpmfusion-free-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=0
+
+[rpmfusion-free-russianfedora-updates]
+name=RPM Fusion for Fedora \$releasever - Free - Updates
+failovermethod=priority
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=build-rpmfusion-free-updates-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=0
+
+[rpmfusion-nonfree-russianfedora]
+name=RPM Fusion for Fedora \$releasever - Non-Free
+failovermethod=priority
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=build-rpmfusion-nonfree-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=0
+
+[rpmfusion-nonfree-russianfedora-updates]
+name=RPM Fusion for Fedora \$releasever - Non-Free - Updates
+failovermethod=priority
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=build-rpmfusion-nonfree-updates-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=0
+
+[russianfedora-fixes]
+name=Russian Fedora for Fedora \$releasever - Fixes
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=fixes-fedora-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-russianfedora-fixes-fedora
+
+[russianfedora-fixes-updates]
+name=Russian Fedora for Fedora \$releasever - Fixes - Updates
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=fixes-fedora-updates-released-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-russianfedora-fixes-fedora
+
+[russianfedora-free]
+name=Russian Fedora for Fedora \$releasever - Free
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=free-fedora-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-russianfedora-free-fedora
+
+[russianfedora-free-updates]
+name=Russian Fedora for Fedora \$releasever - Free - Updates
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=free-fedora-updates-released-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-russianfedora-free-fedora
+[russianfedora-nonfree]
+
+name=Russian Fedora for Fedora \$releasever - Non-Free
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=nonfree-fedora-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-russianfedora-nonfree-fedora
+
+[russianfedora-nonfree-updates]
+name=Russian Fedora for Fedora \$releasever - Non-Free - Updates
+mirrorlist=http://mirrors.rfremix.ru/mirrorlist?repo=nonfree-fedora-updates-released-\$releasever&arch=\$basearch
+enabled=1
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-russianfedora-nonfree-fedora
+"""
+        self._write_content_to_file(filename, repofile_content)
+
+    def create_gnome_cfg(self, filename):
+        gnome_cfg_content = """
+install
+lang ru_RU.UTF-8
+keyboard ru
+%packages
+@admin-tools
+@base
+@base-x
+@core
+@editors
+@fonts
+@gnome-desktop
+@graphical-internet
+@graphics
+@hardware-support
+@java
+@office
+@printing
+@russian-support
+@sound-and-video
+@system-tools
+@text-internet
+%end
+"""
+        self._write_content_to_file(filename, gnome_cfg_content)
+        
+    def create_kde_cfg(self, filename):
+        kde_cfg_content = """
+install
+lang ru_RU.UTF-8
+keyboard ru
+%packages
+@admin-tools
+@base
+@base-x
+@core
+@editors
+@fonts
+@games
+@kde-desktop
+@graphical-internet
+@graphics
+@hardware-support
+@input-methods
+@java
+@office
+@printing
+@russian-support
+@sound-and-video
+@text-internet
+%end
+"""
+        self._write_content_to_file(filename, kde_cfg_content)
+
+    def create_xfce_cfg(self, filename):
+        xfce_cfg_content = """
+install
+lang ru_RU.UTF-8
+keyboard ru
+%packages
+@admin-tools
+@base
+@base-x
+@core
+@editors
+@fonts
+@games
+@xfce-desktop
+@graphical-internet
+@graphics
+@hardware-support
+@input-methods
+@java
+@office
+@printing
+@russian-support
+@sound-and-video
+@text-internet
+%end
+"""
+        self._write_content_to_file(filename, xfce_cfg_content)
+
+    def create_lxde_cfg(self, filename):
+        lxde_cfg_content = """
+install
+lang ru_RU.UTF-8
+keyboard ru
+%packages
+@admin-tools
+@base
+@base-x
+@core
+@editors
+@fonts
+@games
+@lxde-desktop
+@graphical-internet
+@graphics
+@hardware-support
+@input-methods
+@java
+@office
+printing
+@russian-support
+@sound-and-video
+@text-internet
+%end
+"""
+        self._write_content_to_file(filename, lxde_cfg_content)
+
+    def create_tigro_cfg(self, filename):
+        tigro_cfg_content = """
+install
+lang ru_RU.UTF-8
+keyboard ru
+%packages
+@admin-tools
+@base
+@base-x
+@core
+@editors
+@fonts
+@gnome-desktop
+@graphical-internet
+@graphics
+@hardware-support
+@java
+@office
+@printing
+@russian-support
+@sound-and-video
+@system-tools
+@text-internet
+liferea
+gajim
+gwibber
+mail-notification
+ncftp
+goldendict
+k3b-extras-freeworld
+shutter
+lynx
+%end
+
+%post
+if [ -f /etc/sysconfig/keyboard ]; then
+sed -i 's!grp:alt_shift_toggle,grp_led:scroll!grp:rctrl_rshift_toggle,grp_led:scroll!' /etc/sysconfig/keyboard
+fi
+%end
+"""
+        self._write_content_to_file(filename, tigro_cfg_content)
