diff -urN lorax-20.3.orig/share/runtime-install.tmpl lorax-20.3/share/runtime-install.tmpl
--- lorax-20.3.orig/share/runtime-install.tmpl	2013-10-24 23:41:08.671521349 +0400
+++ lorax-20.3/share/runtime-install.tmpl	2013-10-24 23:46:02.031910392 +0400
@@ -72,6 +72,11 @@
 installpkg xorg-x11-server-utils xorg-x11-xauth
 installpkg dbus-x11 metacity gsettings-desktop-schemas
 installpkg nm-connection-editor
+## Connect to VPN
+installpkg NetworkManager-openconnect NetworkManager-openswan
+installpkg NetworkManager-openvpn-gnome NetworkManager-pptp-gnome
+installpkg NetworkManager-vpnc-gnome NetworkManager-l2tp
+
 installpkg at-spi at-spi2-atk pyatspi
 installpkg gnome-python2-gconf
 installpkg gobject-introspection
