diff -urN lorax-0.4.6.orig/share/ramdisk.ltmpl lorax-0.4.6/share/ramdisk.ltmpl
--- lorax-0.4.6.orig/share/ramdisk.ltmpl	2011-08-01 15:43:54.268363890 +0400
+++ lorax-0.4.6/share/ramdisk.ltmpl	2011-08-01 15:44:12.284695643 +0400
@@ -535,7 +535,6 @@
 remove "ModemManager" "/usr/sbin/*"
 remove "ModemManager" "/usr/share/icons/*"
 remove "NetworkManager" "/etc/rc.d/*"
-remove "NetworkManager" "/lib/systemd/*"
 remove "NetworkManager" "/usr/bin/*"
 remove "NetworkManager" "/usr/share/NetworkManager/*"
 remove "NetworkManager" "/usr/share/locale/*/NetworkManager.mo"
