diff -urN lorax-24.18.orig/share/templates.d/99-generic/runtime-cleanup.tmpl lorax-24.18/share/templates.d/99-generic/runtime-cleanup.tmpl
--- lorax-24.18.orig/share/templates.d/99-generic/runtime-cleanup.tmpl	2016-05-20 14:54:16.968455474 +0300
+++ lorax-24.18/share/templates.d/99-generic/runtime-cleanup.tmpl	2016-05-20 14:57:31.568114718 +0300
@@ -38,6 +38,9 @@
                                    /usr/lib/os.release.d/*
 removepkg fedora-release-rawhide
 
+# exclude fedora-releases packages as dnf loses provides
+runcmd sed -i "1a exclude=fedora-release,fedora-release-notes,fedora-release-cloud,fedora-release-server,fedora-release-atomichost,fedora-release-workstation" /etc/dnf/dnf.conf
+
 ## keep enough of shadow-utils to create accounts
 removefrom shadow-utils --allbut /usr/bin/chage /usr/sbin/chpasswd \
                         /usr/sbin/groupadd /usr/sbin/useradd
