diff -urN lorax-18.22.orig/share/runtime-install.tmpl lorax-18.22/share/runtime-install.tmpl
--- lorax-18.22.orig/share/runtime-install.tmpl	2012-11-10 21:11:29.543258819 +0400
+++ lorax-18.22/share/runtime-install.tmpl	2012-11-10 21:12:51.055889734 +0400
@@ -65,6 +65,11 @@
 installpkg xorg-x11-server-utils xorg-x11-xauth
 installpkg dbus-x11 metacity gsettings-desktop-schemas
 installpkg nm-connection-editor
+## Connect to VPN
+installpkg NetworkManager-openconnect NetworkManager-openswan
+installpkg NetworkManager-openvpn NetworkManager-pptp
+installpkg NetworkManager-vpnc
+
 installpkg at-spi at-spi2-atk pyatspi
 installpkg gnome-python2-gconf
 installpkg gobject-introspection
