diff -urN lorax-19.3.orig/share/runtime-cleanup.tmpl lorax-19.3/share/runtime-cleanup.tmpl
--- lorax-19.3.orig/share/runtime-cleanup.tmpl	2013-05-15 16:28:06.387703241 +0400
+++ lorax-19.3/share/runtime-cleanup.tmpl	2013-05-15 16:29:29.261708690 +0400
@@ -29,6 +29,12 @@
 ## anaconda has its own repo files
 removefrom fedora-release --allbut /etc/os-release
 removepkg fedora-release-rawhide
+## remove rfremix packs
+removepkg rpmfusion-nonfree-release rpmfusion-free-release
+removepkg russianfedora-nonfree-release russianfedora-free-release
+removepkg russianfedora-fixes-release
+remove /etc/anaconda.repos.d/*rawhide*.repo
+remove /etc/anaconda.repos.d/rfremix-install-media-dvd.repo
 ## no user accounts = no account management
 removepkg usermode usermode-gtk passwd shadow-utils
 ## no services to turn on/off (keep the /etc/init.d link though)
diff -urN lorax-19.3.orig/share/runtime-install.tmpl lorax-19.3/share/runtime-install.tmpl
--- lorax-19.3.orig/share/runtime-install.tmpl	2013-05-15 16:28:06.395703241 +0400
+++ lorax-19.3/share/runtime-install.tmpl	2013-05-15 16:29:58.189710593 +0400
@@ -140,5 +140,12 @@
 ## yum plugins
 installpkg yum-plugin-fastestmirror yum-langpacks
 
+## RFRemix release packs
+installpkg rpmfusion-nonfree-release
+installpkg rpmfusion-free-release
+installpkg russianfedora-nonfree-release
+installpkg russianfedora-free-release
+installpkg russianfedora-fixes-release
+
 ## actually install all the requested packages
 run_pkg_transaction
