diff -urN lorax-18.29.orig/src/pylorax/treebuilder.py lorax-18.29/src/pylorax/treebuilder.py
--- lorax-18.29.orig/src/pylorax/treebuilder.py	2013-01-11 20:12:49.173041402 +0400
+++ lorax-18.29/src/pylorax/treebuilder.py	2013-01-11 20:13:17.542190500 +0400
@@ -81,7 +81,7 @@
 
     def _install_branding(self):
         release = None
-        for pkg in self.yum.whatProvides('/etc/system-release', None, None):
+        for pkg in self.yum.whatProvides('/etc/rfremix-release', None, None):
             if pkg.name.startswith('generic'):
                 continue
             else:
