diff -urN lorax-22.5.orig/share/runtime-install.tmpl lorax-22.5/share/runtime-install.tmpl
--- lorax-22.5.orig/share/runtime-install.tmpl	2015-03-05 22:57:15.706452439 +0300
+++ lorax-22.5/share/runtime-install.tmpl	2015-03-05 22:57:54.690774800 +0300
@@ -76,6 +76,10 @@
 installpkg xorg-x11-server-utils xorg-x11-xauth
 installpkg dbus-x11 metacity gsettings-desktop-schemas
 installpkg nm-connection-editor
+## Connect to VPN
+installpkg NetworkManager-openconnect NetworkManager-openswan
+installpkg NetworkManager-openvpn-gnome NetworkManager-pptp-gnome
+installpkg NetworkManager-vpnc-gnome NetworkManager-l2tp
 installpkg gobject-introspection
 installpkg librsvg2
 installpkg polkit-desktop-policy
