diff -urN lorax-25.16.orig/share/templates.d/99-generic/runtime-cleanup.tmpl lorax-25.16/share/templates.d/99-generic/runtime-cleanup.tmpl
--- lorax-25.16.orig/share/templates.d/99-generic/runtime-cleanup.tmpl	2016-09-28 12:39:18.742981199 +0300
+++ lorax-25.16/share/templates.d/99-generic/runtime-cleanup.tmpl	2016-09-28 12:40:10.486427557 +0300
@@ -32,6 +32,11 @@
 ## anaconda has its own repo files
 removefrom fedora-release --allbut /etc/os-release /usr/lib/os-release \
                                    /usr/lib/os.release.d/*
+## remove rfremix packs
+removepkg rpmfusion-nonfree-release rpmfusion-free-release
+removepkg russianfedora-nonfree-release russianfedora-free-release
+removepkg russianfedora-fixes-release russianfedora-branding-release
+remove /etc/anaconda.repos.d/russianfedora*rawhide*.repo
 removepkg fedora-release-rawhide
 
 ## keep enough of shadow-utils to create accounts
diff -urN lorax-25.16.orig/share/templates.d/99-generic/runtime-install.tmpl lorax-25.16/share/templates.d/99-generic/runtime-install.tmpl
--- lorax-25.16.orig/share/templates.d/99-generic/runtime-install.tmpl	2016-09-28 12:39:18.742981199 +0300
+++ lorax-25.16/share/templates.d/99-generic/runtime-install.tmpl	2016-09-28 12:40:54.409806455 +0300
@@ -167,6 +167,14 @@
 ## satisfy libnotify's desktop-notification-daemon with the least crazy option
 installpkg notification-daemon
 
+## RFRemix release packs
+installpkg rpmfusion-nonfree-release
+installpkg rpmfusion-free-release
+installpkg russianfedora-nonfree-release
+installpkg russianfedora-free-release
+installpkg russianfedora-fixes-release
+installpkg russianfedora-branding-release
+
 ## Docker enabled boot.iso
 # Not all arches currently have docker
 %if basearch not in ("ppc", "ppc64", "s390"):
