diff -urN lorax-17.9.orig/share/config_files/live/x86/isolinux.cfg lorax-17.9/share/config_files/live/x86/isolinux.cfg
--- lorax-17.9.orig/share/config_files/live/x86/isolinux.cfg	2012-03-10 21:30:23.263657335 +0400
+++ lorax-17.9/share/config_files/live/x86/isolinux.cfg	2012-03-10 21:30:45.303071789 +0400
@@ -62,6 +62,21 @@
   kernel vmlinuz
   append initrd=initrd.img @ROOT@ quiet liveimg rd.luks=0 rd.md=0 rd.dm=0
 menu separator # insert an empty line
+label install
+  menu label ^Install @PRODUCT@
+  text help
+	Install system to your computer
+  endtext
+  kernel vmlinuz
+  append initrd=initrd.img @ROOT@ quiet liveimg liveinst rd.luks=0 rd.md=0 rd.dm=0
+label ram
+  menu label Start @PRODUCT@ into ^RAM
+  text help
+	If you have more then 2GB of RAM you can start
+	@PRODUCT@ into the memory.
+  endtext
+  kernel vmlinuz
+  append initrd=initrd.img @ROOT@ quiet liveimg live_ram rd.luks=0 rd.md=0 rd.dm=0
 # utilities submenu
 menu begin ^Troubleshooting
   menu title Troubleshooting
diff -urN lorax-17.9.orig/share/runtime-install.tmpl lorax-17.9/share/runtime-install.tmpl
--- lorax-17.9.orig/share/runtime-install.tmpl	2012-03-10 21:30:23.261657297 +0400
+++ lorax-17.9/share/runtime-install.tmpl	2012-03-10 21:30:45.304071809 +0400
@@ -93,6 +93,7 @@
 installpkg fedora-gnome-theme fedora-icon-theme
 installpkg ${product.name}-logos
 installpkg ${product.name}-release
+installpkg ${product.name}-quick-install
 
 ## debugging/bug reporting tools
 installpkg gdb-gdbserver
diff -urN lorax-17.9.orig/share/runtime-postinstall.tmpl lorax-17.9/share/runtime-postinstall.tmpl
--- lorax-17.9.orig/share/runtime-postinstall.tmpl	2012-03-10 21:30:23.264657354 +0400
+++ lorax-17.9/share/runtime-postinstall.tmpl	2012-03-10 21:32:44.557314376 +0400
@@ -29,6 +29,8 @@
 
 ## move_repos()
 move etc/yum.repos.d etc/anaconda.repos.d
+remove etc/anaconda.repos.d/fedora*repo
+remove etc/anaconda.repos.d/rfremix-install-media-dvd.repo 
 
 ## Configure systemd to start anaconda
 remove etc/systemd/system/default.target
