diff -urN lorax-18.21.orig/share/config_files/x86/isolinux.cfg lorax-18.21/share/config_files/x86/isolinux.cfg
--- lorax-18.21.orig/share/config_files/x86/isolinux.cfg	2012-10-29 19:28:42.628307371 +0400
+++ lorax-18.21/share/config_files/x86/isolinux.cfg	2012-10-29 19:29:20.893369310 +0400
@@ -67,6 +67,46 @@
   append initrd=initrd.img @ROOT@ quiet rd.live.check
 menu separator # insert an empty line
 # utilities submenu
+menu begin ^Quick install
+  menu title Quick install
+label gnome
+  menu indent count 5
+  menu label Install GNOME Desktop (in Russian)
+  text help
+        Quick installation of GNOME Desktop
+  endtext
+  kernel vmlinuz
+  append initrd=initrd.img @ROOT@ quiet rd.luks=0 rd.md=0 rd.dm=0 inst.ks=hd:/dev/mapper/live-rw:/gnome.cfg
+label kde
+  menu indent count 5
+  menu label Install KDE (in Russian)
+  text help
+        Quick installation of Plasma
+  endtext
+  kernel vmlinuz
+  append initrd=initrd.img @ROOT@ quiet rd.luks=0 rd.md=0 rd.dm=0 inst.ks=hd:/dev/mapper/live-rw:/kde.cfg
+label xfce
+  menu indent count 5
+  menu label Install XFCE Desktop (in Russian)
+  text help
+        Quick installation of XFCE Desktop
+  endtext
+  kernel vmlinuz
+  append initrd=initrd.img @ROOT@ quiet rd.luks=0 rd.md=0 rd.dm=0 inst.ks=hd:/dev/mapper/live-rw:/xfce.cfg
+label lxde
+  menu indent count 5
+  menu label Install LXDE Desktop (in Russian)
+  text help
+        Quick installation of LXDE Desktop
+  endtext
+  kernel vmlinuz
+  append initrd=initrd.img @ROOT@ quiet rd.luks=0 rd.md=0 rd.dm=0 inst.ks=hd:/dev/mapper/live-rw:/lxde.cfg
+menu separator # insert an empty line
+menu separator # insert an empty line
+label returntomain
+  menu label Return to ^main menu.
+  menu exit
+menu end
 menu begin ^Troubleshooting
   menu title Troubleshooting
 label vesa
diff -urN lorax-18.21.orig/share/runtime-cleanup.tmpl lorax-18.21/share/runtime-cleanup.tmpl
--- lorax-18.21.orig/share/runtime-cleanup.tmpl	2012-10-29 19:28:42.628307371 +0400
+++ lorax-18.21/share/runtime-cleanup.tmpl	2012-10-29 19:29:20.894369312 +0400
@@ -26,6 +26,12 @@
 removepkg checkpolicy selinux-policy libselinux-utils
 ## anaconda has its own repo files
 removepkg fedora-release fedora-release-rawhide
+## remove rfremix packs
+removepkg rpmfusion-nonfree-release rpmfusion-free-release
+removepkg russianfedora-nonfree-release russianfedora-free-release
+removepkg russianfedora-fixes-release
+remove /etc/anaconda.repos.d/*rawhide*.repo
+remove /etc/anaconda.repos.d/rfremix-install-media-dvd.repo
 ## no user accounts = no account management
 removepkg usermode usermode-gtk passwd shadow-utils
 ## no services to turn on/off (keep the /etc/init.d link though)
diff -urN lorax-18.21.orig/share/runtime-install.tmpl lorax-18.21/share/runtime-install.tmpl
--- lorax-18.21.orig/share/runtime-install.tmpl	2012-10-29 19:28:42.630307376 +0400
+++ lorax-18.21/share/runtime-install.tmpl	2012-10-29 19:29:20.894369312 +0400
@@ -126,5 +126,13 @@
 ## yum plugins
 installpkg yum-plugin-fastestmirror
 
+## RFRemix release packs
+installpkg rpmfusion-nonfree-release
+installpkg rpmfusion-free-release
+installpkg russianfedora-nonfree-release
+installpkg russianfedora-free-release
+installpkg russianfedora-fixes-release
+installpkg rfremix-quick-install
+
 ## actually install all the requested packages
 run_pkg_transaction
diff -urN lorax-18.21.orig/src/pylorax/__init__.py lorax-18.21/src/pylorax/__init__.py
--- lorax-18.21.orig/src/pylorax/__init__.py	2012-10-29 19:28:42.631307378 +0400
+++ lorax-18.21/src/pylorax/__init__.py	2012-10-29 19:29:20.895369314 +0400
@@ -279,7 +279,7 @@
 
         logger.info("rebuilding initramfs images")
         dracut_args=["--xz", "--add", "anaconda pollcdrom",
-                     "--install", "/.buildstamp"]
+                     "--install", "/.buildstamp ./*.cfg"]
         treebuilder.rebuild_initrds(add_args=dracut_args)
 
         logger.info("populating output tree and building boot images")
