diff -urN lorax-22.8.orig/share/runtime-cleanup.tmpl lorax-22.8/share/runtime-cleanup.tmpl
--- lorax-22.8.orig/share/runtime-cleanup.tmpl	2015-04-01 18:40:05.452281304 +0300
+++ lorax-22.8/share/runtime-cleanup.tmpl	2015-04-01 18:41:09.919905534 +0300
@@ -29,6 +29,11 @@
 ## anaconda has its own repo files
 removefrom fedora-release --allbut /etc/os-release /usr/lib/os-release \
                                    /usr/lib/os.release.d/*
+## remove rfremix packs
+removepkg rpmfusion-nonfree-release rpmfusion-free-release
+removepkg russianfedora-nonfree-release russianfedora-free-release
+removepkg russianfedora-fixes-release russianfedora-branding-release
+remove /etc/anaconda.repos.d/russianfedora*rawhide*.repo
 removepkg fedora-release-rawhide
 ## no user accounts = no account management
 removepkg usermode usermode-gtk passwd shadow-utils
diff -urN lorax-22.8.orig/share/runtime-install.tmpl lorax-22.8/share/runtime-install.tmpl
--- lorax-22.8.orig/share/runtime-install.tmpl	2015-04-01 18:40:05.447281178 +0300
+++ lorax-22.8/share/runtime-install.tmpl	2015-04-01 18:41:52.200973628 +0300
@@ -156,5 +156,13 @@
 ## yum plugins
 installpkg yum-plugin-fastestmirror yum-langpacks
 
+## RFRemix release packs
+installpkg rpmfusion-nonfree-release
+installpkg rpmfusion-free-release
+installpkg russianfedora-nonfree-release
+installpkg russianfedora-free-release
+installpkg russianfedora-fixes-release
+installpkg russianfedora-branding-release
+
 ## actually install all the requested packages
 run_pkg_transaction
