diff -urN lorax-0.4.6.orig/share/ramdisk.ltmpl lorax-0.4.6/share/ramdisk.ltmpl
--- lorax-0.4.6.orig/share/ramdisk.ltmpl	2011-08-02 23:24:40.495711402 +0400
+++ lorax-0.4.6/share/ramdisk.ltmpl	2011-08-02 23:25:39.771249550 +0400
@@ -47,6 +47,11 @@
 
 ## other required packages
 install "NetworkManager-gnome"
+install "NetworkManager-openconnect"
+install "NetworkManager-openswan"
+install "NetworkManager-openvpn"
+install "NetworkManager-pptp"
+install "NetworkManager-vpnc"
 install "at-spi"
 install "at-spi-python"
 install "at-spi2-atk"
@@ -139,6 +144,7 @@
 install "specspo"
 install "strace"
 install "systemd-sysvinit"
+install "sysvinit-tools"
 install "tigervnc-server-minimal"
 install "tigervnc-server-module"
 install "udev"
