diff -urN lorax-18.23.orig/docs/fedora-livemedia.ks lorax-18.23/docs/fedora-livemedia.ks
--- lorax-18.23.orig/docs/fedora-livemedia.ks	2012-12-10 00:29:01.326101972 +0400
+++ lorax-18.23/docs/fedora-livemedia.ks	2012-12-10 00:32:25.160078134 +0400
@@ -385,6 +385,7 @@
 -smartmontools
 grub2-efi
 grub2
-shim
+shim-signed
+shim-unsigned
 
 %end
diff -urN lorax-18.23.orig/share/runtime-install.tmpl lorax-18.23/share/runtime-install.tmpl
--- lorax-18.23.orig/share/runtime-install.tmpl	2012-12-10 00:29:01.318102009 +0400
+++ lorax-18.23/share/runtime-install.tmpl	2012-12-10 00:30:51.333539701 +0400
@@ -30,7 +30,7 @@
     installpkg kernel-PAE gpart
 %endif
 %if basearch in ("i386", "x86_64"):
-    installpkg shim grub2-efi efibootmgr biosdevname memtest86+
+    installpkg shim-signed shim-unsigned grub2-efi efibootmgr biosdevname memtest86+
 %endif
 %if basearch in ("ppc", "ppc64"):
     installpkg yaboot fbset hfsutils kernel-bootwrapper ppc64-utils
